packages/block-library/src/embed/util.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { common, others } from './core-embeds';
+import { DEFAULT_EMBED_BLOCK } from './constants';
+
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+export const matchesPatterns = ( url, patterns = [] ) => {
```

Comment:
It's great having these functions in their own file. Would be good to add jsdoc comments and add some unit tests for them.